Dear/Dear/Cell/CheeringTextTableViewCell.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  CheeringTextTableViewCell.swift
+//  Dear
+//
+//  Created by Jeon Jimin on 2022/07/19.
+//
+
+import UIKit
+
+class CheeringTextTableViewCell: UITableViewCell {
+    @IBOutlet weak var CheeringTextTitleLabel: UILabel!
```

Comment:
```suggestion
    @IBOutlet weak var cheeringTextTitleLabel: UILabel!
```

Swift에서는 property는 low camel case로 하는편이에요 🐫

Reply:
넵 반영해서 수정하도록 하겠습니다!